static/app/components/core/slideOverPanel/slideOverPanel.mdx

```diff
@@ -30,7 +30,7 @@ The `SlideOverPanel` component is a panel that can appear on the right, left, or
 ```jsx
```

Comment:
How about just `open`?
```suggestion
<SlideOverPanel open={isPanelOpen} slidePosition="right">
```

`slidePosition` also feels bad. It could just be `position` right since we are tweaking the global position?